pkg/storkctl/migration.go

```diff
@@ -180,19 +180,26 @@ func newActivateMigrationsCommand(cmdFactory Factory, ioStreams genericclioption
 				}
 			}
 			for _, ns := range migrationScheduleNamespaces {
-				migrSched, err := storkops.Instance().ListMigrationSchedules(ns)
+				migrationSchedules, err := storkops.Instance().ListMigrationSchedules(ns)
 				if err != nil {
 					util.CheckErr(err)
 					return
 				}
-				for _, migr := range migrSched.Items {
-					migr.Status.ApplicationActivated = true
-					_, err := storkops.Instance().UpdateMigrationSchedule(&migr)
+				for _, migrSched := range migrationSchedules.Items {
+					isMigrSchedRelevant, err := isMigrationScheduleRelevant(migrSched, activationNamespaces)
 					if err != nil {
 						util.CheckErr(err)
 						return
 					}
-					printMsg(fmt.Sprintf("Set the ApplicationActivated status in the MigrationSchedule %v/%v to true", migr.Namespace, migr.Name), ioStreams.Out)
+					if isMigrSchedRelevant {
+						migrSched.Status.ApplicationActivated = true
+						_, err := storkops.Instance().UpdateMigrationSchedule(&migrSched)
+						if err != nil {
+							util.CheckErr(err)
+							return
+						}
+						printMsg(fmt.Sprintf("Set the ApplicationActivated status in the MigrationSchedule %v/%v to true", migrSched.Namespace, migrSched.Name), ioStreams.Out)
```

Comment:
Nit: Setting the ApplicationActivated status

Reply:
will address